Android-Studio-Plugin/build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 }
 
 group 'test'
-version '1.7-SNAPSHOT'
```

Comment:
I tried to create releases retroactively, but the previous build was pushed to the repository in batches.
Since there was no other release management, it makes sense to reset the counter to version 1.0.
Version 1.0 represents the project state done by Christian.
Now, I will be incrementing the version and adding a release as the issues are resolved (if it makes changes other than structure, such as this PR)